src/Cppyy.h

```diff
@@ -321,7 +325,8 @@ namespace Cppyy {
     TCppType_t  GetType(const std::string& name, bool enable_slow_lookup = false);
     CPPYY_IMPORT
     bool AppendTypesSlow(const std::string &name,
-                         std::vector<Cpp::TemplateArgInfo>& types);
+                         std::vector<Cpp::TemplateArgInfo>& types,
+                         bool no_reference=false);
```

Comment:
Note to myself: This change is not required. Anyway if the CI runs it will fail.